src/app/app.module.ts

```diff
@@ -121,6 +125,7 @@ const routes: Routes = [
   imports: [BrowserModule, FormsModule,
     HttpClientModule,
     HttpClientJsonpModule,
+    MatTabsModule,
```

Comment:
Those two new modules aren't used anywhere in the main app, so here they'll just make the compilation longer (or the bundle larger).